frontend/ui/impl/build.gradle

```diff
@@ -0,0 +1,23 @@
+lombok()
+slf4jLambdaApi()
+vavr()
+
+dependencies {
+  implementation project(':binding')
+  implementation project(':branchLayout:api')
+  implementation project(':backend:api')
+  implementation project(':frontend:compatibilityBridge')
+  implementation project(':frontend:defs')
+  implementation project(':frontend:dataKeys')
+  implementation project(':frontendGraph:api')
```

Comment:
`frontend:graph:api`?

Reply:
so maybe put every frontend module inside frontend directory ? include frontend actions ?

Reply:
Yeah, exactly... to be further revamped within #796 (but even then, I think all modules will stay under `frontend/`)